src/test/java/com/github/onsdigital/dp/interactives/api/InteractivesAPIClientTest.java

```diff
@@ -168,4 +169,40 @@ void handingIncorrectHostnameProvided() {
 
         assertNotNull(e.getCause());
     }
+
+    @Test
+    void deleteInteractiveHandingAuthorizationFailure() {
+        try (MockWebServer server = new MockWebServer()) {
+            server.enqueue(new MockResponse().setResponseCode(HttpStatus.SC_UNAUTHORIZED));
+
+            HttpUrl url = server.url("");
+
+            InteractivesAPIClient client = new InteractivesAPIClient(url.toString(), TOKEN);
+
+            assertThrows(UnauthorizedException.class, () -> {
+                client.deleteInteractive(INTERACTIVE_ID);
+            });
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
```

Comment:
Since this is a test, I think we don't need to catch the exception at all and could just throw it